gradle/codequality/suppressions.xml

```diff
@@ -8,4 +8,6 @@
         files=".*[\\/]protogen[\\/].*"/>
     <suppress checks="JavadocPackage"
         files=".*[\\/]test[\\/].*|.*[\\/]java-proto[23][\\/].*[\\/]metadata[\\/].*"/>
+    <suppress checks="Header"
+              files=".*[\\/\]org[\\/]apache[\\/]lucene[\\/]codecs[\\/]lucene84[\\/]LuceneOptimizedPostingsReader\.java"/>
```

Comment:
```suggestion
        files=".*[\\/]org[\\/]apache[\\/]lucene[\\/]codecs[\\/]lucene84[\\/]LuceneOptimizedPostingsReader\.java"/>
```

It looks like there's an extra `/` before `org`, which is what caused the gradle check style build to fail